app/sose2019/templates/index.html

```diff
@@ -143,11 +143,11 @@ <h2 class="panel-title">Sonstiges</h3>
 	  {{ form_group(form.nrwticket) }}
           {{ form_group(form.tshirt, 'Im Anmeldepreis enthalten.') }}
           {{ form_group(form.addtshirt, 'Zusätzliche Teilnehmershirts, Größe wie bei diesen angegeben, Preis 10€/Stück. Verbindliche Kaufzusage.') }}
-	  {{ checkbox(form.bottle, 'ca 10€, verbindliche Kaufzusage.') }}
-	  {{ checkbox(form.bottle, 'maximal 5€, verbindliche Kaufzusage. (Mindestbestellmenge 100 Stück)') }}
-	  {{ form_group(form.quant, '2€ pro Bogen, Bestellung in Anzahl Bögen.') }}
+	  {{ checkbox(form.bottle, 'Wasserflaschen mit Zapflogo. maximal 10€ (Mindestbestellmenge 50 Stück)') }}
+	  {{ checkbox(form.bottle, 'Bierkrüge, maximal 5€. (Mindestbestellmenge 100 Stück)') }}
+	  {{ form_group(form.quant, 'Quanten-Aufkleder, 2€ pro Bogen.') }}
 	  {{ form_group(form.zaepfchen, 'Wenn du angibst einen Mentor zu wollen, erklärst du dich damit einverstanden, dass wir deine E-mail Adresse an deinen Mentor weitergegeben.') }}
-	  {{ checkbox(form.mentor) }}
+	  {{ checkbox(form.mentor, 'Hast du die Hallenordnung gelesen?') }}
```

Comment:
sollte das nicht vielleicht in die nächste Zeile?

Reply:
ehm ja, ups